nativescript-angular/directives/dialogs.ts

```diff
@@ -111,6 +121,13 @@ export class ModalDialogService {
         });
     }
 
+    public closeModal() {
```

Comment:
Can you remove this method. After locally testing and reviewing this PR I discovered a bug (https://github.com/NativeScript/NativeScript/issues/7050) that when fixed will resolve this issue you are working around with this code so it is not needed.